hash/vsmile_cart.xml

```diff
@@ -629,6 +630,21 @@ V.Smile Smartbook Smartidges (need a Smartbook touch tablet connected to a regul
 		</part>
 	</software>
 
+	<software name="aladdinuk" cloneof="aladdinit">
+		<description>Disney's Aladdin - Aladdin's Wonders of the World (UK)</description>
+		<year>2005</year>
+		<publisher>VTech</publisher>
+		<info name="serial" value="80-092203(UK)" />
+		<part name="cart" interface="vsmile_cart">
+			<feature name="slot" value="vsmile_rom" />
+			<feature name="cart_type" value="lilac" />
+			<feature name="u1" value="" />
+			<dataarea name="rom" size="8388608">
+				<rom name="52-92203- Disney Aladdin - Wonders of the World (UK).bin" size="8388608" crc="244eb37a" sha1="c8131f488e563ef1a7f045ef3601670d02ca404e" />
+			</dataarea>
+		</part>
+	</software>
+
```

Comment:
Should this be the new parent?  We usually go for English-language parents.

Reply:
Yes!
I was thinking on using the VTech precedence (USA -> Italy -> Netherlands -> UK -> Germany...), but makes more sense to follow the usual "English first".

Thanks!!!